src/index.html

```diff
@@ -11,6 +15,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
```

Comment:
```suggestion
      action="#"
```
try to change and test what is the difference
or you can use the link from the previous task

Reply:
done